### PR TITLE
Cpp/tests/ci

### PR DESCRIPTION
**Pull request by Shylock-Hg**
Follow #21

### CODE REVIEW

**Review by yixinglu — DISMISSED** [review]

File: .github/workflows/cpp_test.yml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        run: cmake --build build/ -j $(nproc)
+      - name: Nebula Server
+        run: |
+          sudo curl -sSL https://get.docker.com/ | sh
+          pip3 install setuptools
+          pip3 install docker-compose
+          docker network ls
+          mkdir tmp
+          pushd tmp
+          git clone -b v2-preview-external_network https://github.com/Shylock-Hg/nebula-docker-compose.git
```
Comment (laura-ding) [inline .github/workflows/cpp_test.yml:100]: why use your repo?

File: cpp/package/package.sh [hunk 2]
```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 vesoft inc. All rights reserved.
```
Comment (laura-ding) [inline cpp/package/package.sh]: 2020

File: cpp/package/package.sh [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+#
+#  Package nebula as deb/rpm package
+#
+# introduce the args
+#   -v: The version of package, the version should be match tag name, default value is the `commitId`
+#   -n: Package to one or multi packages, `ON` means one package, `OFF` means multi packages, default value is `ON`
+#   -s: Whether to strip the package, default value is `FALSE`
+#   -a: Whether to disable the cxx11 abi, default value is `OFF`
+#
+# usage: ./package.sh -v <version> -n <ON/OFF> -s <TRUE/FALSE> -b <BRANCH>
```
Comment (laura-ding) [inline cpp/package/package.sh:17]: here is no need `-n`, the enable to package to one or multi files

File: .github/workflows/cpp_test.yml (hunk 1 above)
Comment (Shylock-Hg) [reply .github/workflows/cpp_test.yml:100]: I try the external network

File: cpp/package/package.sh (hunk 2 above)
Comment (Shylock-Hg) [reply cpp/package/package.sh]: Fixed.

File: cpp/package/package.sh (hunk 3 above)
Comment (Shylock-Hg) [reply cpp/package/package.sh:17]: Don't plan to modify the script too much.

**Comment by Shylock-Hg** [comment]
Reopen in https://github.com/vesoft-inc/nebula-cpp/pull/3

**Pull request closed without merging**